requirements.txt

```diff
@@ -9,7 +9,6 @@ django-ses-backend==0.1.1
 djmail==0.11.0
 ez-setup==0.9
 futures==3.0.3
-gunicorn==19.7.1
```

Comment:
Hi @Jak-Sidious kindly add the gunicorn dependency. It's useful when deploying the app to heroku.
Cc: @VincentHokie